### ISSUE TITLE
Please apply the license correctly

### ISSUE THREAD

**Issue by HiPhish** [Ferada/cl-mock#8]
Hello,

the README says that the project is licensed under the GNU Affero GPL v3+, but that is not enough to properly license the project. You need a `LICENSE` or `COPYING` file at the root of the repository with the full text of the license, and you need to add the copyright- and  license notice to each file. See here for instructions:

https://www.gnu.org/licenses/gpl-howto.html

And while at it, it would be great if you were to replace the `:license` field in the ASDF file with the [SPDX identiefier](https://spdx.org/licenses/AGPL-3.0-or-later.html) (`:license "AGPL-3.0-or-later"`) and add an [SPDX license identifier](https://spdx.dev/ids/) at the top of each source file (`;;;;  SPDX-License-Identifier: AGPL-3.0-or-later`)

I have done the same thing in my own project [cl-hssg](https://gitlab.com/HiPhish/cl-hssg) which uses the same license, if you want to see an example.

**Comment by Ferada** [comment]
I'm good thank you, I'll accept PRs for it though.

**Comment by HiPhish** [comment]
> I'm good thank you, I'll accept PRs for it though.

Fair enough. What should I put into the copyright statement? Years? Authors?
